packages/vote/statement/opinion.go

```diff
@@ -0,0 +1,125 @@
+package statement
+
+import (
+	"sort"
+	"strconv"
+
+	"github.com/iotaledger/goshimmer/packages/vote"
+	"github.com/iotaledger/hive.go/cerrors"
+	"github.com/iotaledger/hive.go/marshalutil"
+	"github.com/iotaledger/hive.go/stringify"
+	"golang.org/x/xerrors"
+)
+
+const (
+	// OpinionLength defines the opinion length in bytes.
+	OpinionLength = 2
+)
+
+// region Opinion /////////////////////////////////////////////////////////////////////////////////////////////////////
+
+// Opinion holds the opinion at a specific round.
+type Opinion struct {
+	Value vote.Opinion
+	Round uint8
+}
+
+// Bytes returns a marshaled version of the opinion.
+func (o Opinion) Bytes() (bytes []byte) {
+	bytes = make([]byte, OpinionLength)
+
+	// initialize helper
+	marshalUtil := marshalutil.New(bytes)
+
+	marshalUtil.WriteByte(byte(o.Value))
+	marshalUtil.WriteUint8(o.Round)
+
+	return bytes
```

Comment:
I guess I missed this in the last review. Would be great to change it to the proper usage like for the others